cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -110,6 +110,51 @@ namespace aspect
   }
 
   /**
+   * This function retrieves the unit support points (in the unit cell) for the current element
+   * The DGP elements used when 'set Use locally conservative discretization = true' is enabled do
+   * not have support points. If these elements are in use, a ficticious support point at the cell
+   * center is returned. This is necessary for the internal velocities to be constrained as desired.
+   */
+  template <int dim>
+  std::vector< Point<dim> >
+  get_unit_support_points_for_velocity(const SimulatorAccess<dim> &simulator_access,
+                                       const bool use_locally_conservative_discretization)
+  {
+    std::vector< Point<dim> > unit_support_points;
+    if ( !use_locally_conservative_discretization )
+      {
+        unit_support_points = simulator_access.get_fe().get_unit_support_points();
```

Comment:
or, more easily,
```
  return simulator_access.get_fe().get_unit_support_points();
```
Then you can move the declaration of the `unit_support_points` variable into the `else` block, along with the `return` statement at the bottom of the function.